awx/conf/signals.py

```diff
@@ -37,6 +38,13 @@ def handle_setting_change(key, for_delete=False):
             enter=not bool(for_delete),
         )
 
+    # enabled/disable analytics sys job schedule with setting value
```

Comment:
I understand the desire for backwards compatibility, but it seems a little odd to me that we'd have manual code to keep both a setting and a `SystemJobTemplate` enablement in sync.  What happens if somebody goes and disables the `SystemJobTemplate`?  It seems like `settings.INSIGHTS_TRACKING_STATE` wouldn't be affected?

Reply:
That sounds reasonable.  If @wenottingham agrees with this approach, I will make it so.

Reply:
Is it possible to just have the SystemJobTemplate noop/skip if the setting is off?

Reply:
@wenottingham that would similarly require a signal.

Reply:
How would it require a signal for the management command to just check the setting and do nothing if not set?

Reply:
@wenottingham we already do that.  The issue with that is that it doesn't prevent the job from being run by the schedule, so the system job will periodically attempt to run, fail, and add noise to the logs.  But it is an option.